kingpin/actors/aws/settings.py

```diff
@@ -31,3 +31,4 @@
 # During tests, we mock these out to blank strings to prevent these calls.
 AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID", None)
 AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY", None)
+AWS_SESSION_TOKEN = os.getenv("AWS_SESSION_TOKEN", None)
```

Comment:
This may be obvious, but I’m new to this area of the code and trying to understand. From a design point of view, if we didn’t parse these in settings, and depended on boto3’s handling, would that work? It feels like we are duplicating logic already in the SDK?

Reply:
Yes, exactly what my approval comment pointed out as well. However, I was not 100% sure so the safe thing to do is just do the same env var override that we were doing before, but just capture this missed env var (which lead to some rewrite of the `if` statement in the beginning of the `aws/base.py` code.)